bin/review-rot

```diff
@@ -71,23 +56,134 @@ def main(state_, value, duration, ssl_verify=None):
                     git_service.request_reviews(
                         user_name=res.get('user_name'),
                         repo_name=res.get('repo_name'),
-                        state_=state_,
-                        value=value,
-                        duration=duration,
+                        state_=arguments.get('state'),
+                        value=arguments.get('value'),
+                        duration=arguments.get('duration'),
                         token=item.get('token'),
                         host=item.get('host'),
-                        ssl_verify=ssl_verify,
+                        ssl_verify=arguments.get('ssl_verify', False),
                     )
                 )
 
     # Now, with all results in place, sort them and print
     sorted_results = sorted(
         results,
         key=operator.attrgetter('time'),
-        reverse=args.reverse,
+        reverse=arguments.get('reverse'),
     )
     for result in sorted_results:
-        print(result.format(style=args.format))
+        print(result.format(style=arguments.get('format', 'oneline')))
+
+
+def load_config_file():
+    """
+       Loads the configuration file from the user's home directory.
+       Returns:
+           config(dict): Returns the configurations
+       """
+    home = expanduser('~')
+    path_to_config = os.path.join(home, '.reviewrot.yaml')
+    if not os.path.exists(path_to_config):
+        raise RuntimeError("No config '.reviewrot.yaml' provided. Please"
+                           " specify it in home directory: %s" % home)
+
+    config = load_ordered_config(path_to_config)
+    if isinstance(config, list):
+        # convert to new format
+        config = dict(git_services=config)
+        answer = str(raw_input("Would you like to rewrite '.reviewrot.yaml"
+                               "' in new format [Y/n] : ")).lower().strip()
+        if answer == 'y':
+            # Take the backup of configuration file and
+            # save the configurations in new format
+            backup_path = path_to_config + '.backup'
+            log.info("Creating back up at " + backup_path)
+            copyfile(path_to_config, backup_path)
+            log.info("Rewriting %r in new format!" % path_to_config)
+            with open(path_to_config, 'w') as f:
+                f.write(yaml.dump(config, default_flow_style=False))
+                f.write("\narguments:\n")
```

Comment:
Instead of manually constructing yaml, why not just set `arguments` to `null`?

```python
config = dict(
    git_services=config,
    arguments=None
)
```